lib/plug/static.ex

```diff
@@ -77,6 +77,10 @@ defmodule Plug.Static do
 
     * `:headers` - other headers to be set when serving static assets.
 
+    * `:mime_types` - custom MIME type mapping. As a map with filename as key
```

Comment:
it should be called content_types as this is effectively the name of the header in the response.